deployment/chainloop/values.yaml

```diff
@@ -154,16 +154,10 @@ controlplane:
   ## TLS configuration
   ##
   tls:
-    ## @param controlplane.tls.existingSecret Existing secret with TLS certificates. The secret must contains 2 keys: tls.crt and tls.key respectively containing the certificate and private key.
-    ## NOTE: When it's set it will disable secret creation.
+    ## @param controlplane.tls.existingSecret Existing secret name containing TLS certificate to be used by the controlplane grpc server
+    ## NOTE: When it's set it will disable secret creation. The secret must contains 2 keys: tls.crt and tls.key respectively containing the certificate and private key.
     ##
-    existingSecret: ""
-
-  ## @param controlplane.tlsConfig.secret.name name of a secret containing TLS certificate to be used by the controlplane grpc server.
-  tlsConfig:
-    secret:
-      # the secret must contains 2 keys: tls.crt and tls.key respectively containing the certificate and private key.
-      name: ""
+    existingSecret: "random"
```

Comment:
remove "random" since the other one doesn't have it and it's not meant to be random is it?